### PR TITLE
base.yaml updates for openSUSE (3 of 6)

### PR DESCRIPTION
**Pull request by neotinker**
This is a continuation of the base.yaml updates for openSUSE
#29494 #29640

These keys are not necessarily related to each other. I'm just processing these commits alphabetically and breaking them up into groups of approx. 10.

https://software.opensuse.org/package/libcap
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2:Update/patchinfo.16019/standard
https://software.opensuse.org/package/libdw1
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/elfutils/standard
https://software.opensuse.org/package/libftdi1
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/libftdi1/standard
https://software.opensuse.org/package/gdal
https://build.opensuse.org/package/show/openSUSE%3ALeap%3A15.2/gdal
https://software.opensuse.org/package/gflags
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/gflags/standard
https://software.opensuse.org/package/glew
https://build.opensuse.org/package/show/openSUSE%3ALeap%3A15.2/glew
https://software.opensuse.org/package/google-glog
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/google-glog/standard
https://software.opensuse.org/package/libgpgme11
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2:Update/patchinfo.16131/standard
https://software.opensuse.org/package/libjpeg8
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2:Update/patchinfo.13954/standard
https://software.opensuse.org/package/libjsoncpp19
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/jsoncpp/standard

### CODE REVIEW

File: rosdep/base.yaml [hunk 1]
```diff
@@ -3040,6 +3044,7 @@ libglew-dev:
   gentoo: [media-libs/glew]
   nixos: [glew]
   openembedded: [glew@openembedded-core]
+  opensuse: [glew-devel]
```
Comment (tfoote) [inline rosdep/base.yaml:3047]: The main link didn't find the devel version but it appears to be in the spec file. https://build.opensuse.org/package/view_file/openSUSE:Leap:15.2/glew/glew.spec?expand=1

File: rosdep/base.yaml [hunk 2]
```diff
@@ -2988,7 +2991,7 @@ libgconf2:
 libgdal-dev:
   debian: [libgdal-dev]
   nixos: [gdal]
-  opensuse: [gdal]
+  opensuse: [gdal-devel]
```
Comment (tfoote) [inline rosdep/base.yaml:2994]: I found a reference to the devel version in the spec file here too: https://build.opensuse.org/package/view_file/openSUSE:Leap:15.2/gdal/gdal.spec?expand=1

**Review by tfoote — APPROVED** [review]

**Comment by tfoote** [comment]
Oops sorry, I missed that @mabelzhang hadn't gotten to this one too. Can you check posthoc @mabelzhang ?

**Comment by mabelzhang** [comment]
Checked all. Thank you guys for the links!

File: rosdep/base.yaml (hunk 1 above)
Comment (neotinker) [reply rosdep/base.yaml:3047]: I guess I accidentally posted just the package view link. I've been posting the binary repository links attached to the package which for glew would have been ...

https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/glew/standard

File: rosdep/base.yaml (hunk 2 above)
Comment (neotinker) [reply rosdep/base.yaml:2994]: I did the same thing for glew
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/glew/standard

**Pull request merged**